browser-support.js

```diff
@@ -325,9 +342,23 @@ class BrowserSupport extends HTMLElement {
 
     if (value.status === 'newly') {
       const progress = this.#monthsSince(date);
-      this.#ui.progress.innerHTML = `${progress} months`;
+      this.#ui.progress.innerHTML = `in ${30 - progress} months`;
       this.#ui.progress.value = progress;
 
+      const highDate = new Date(date.setMonth(date.getMonth() + 30));
```

Comment:
This maybe has a slight bug- if `date` is August 31, this sets the date to February 31, or March 3. I'm not sure there's a better way of doing that though? But really, that's only off by 0.3%, so /shrug?